src/functions/template-tags/general.php

```diff
@@ -704,3 +704,22 @@ function tribe_is_frontend() {
 		return (bool) apply_filters( 'tribe_doing_frontend', false );
 	}
 }
+
+if ( ! function_exists( 'tribe_set_time_limit' ) ) {
+	/**
+	 * Wrapper for set_time_limit to suppress errors
+	 *
+	 * @since 4.7.12
+	 *
+	 * @param int $limit Time limit.
+	 */
+	function tribe_set_time_limit( $limit = 0 ) {
+		if (
```

Comment:
Invert the conditional and `return false;`, and after the conditional use `return @set_time_limit...`